### PR TITLE
[ot-glyph] More on testing

### PR DESCRIPTION
**Pull request by ebraminio**
Khaled, please check if it looks good

### CODE REVIEW

File: test/api/fonts/README [hunk 1]
```diff
@@ -3,3 +3,5 @@ cmap-format12-only files created by ttx & remove all other cmap entries
 Inconsolata-Regular.abc.widerc.ttf has the hmtx width of "c" set to 600; everything else is 500. Subsetting out c should reduce numberOfHMetrics to 1.
 
 chromacheck-* fonts are from https://github.com/RoelN/ChromaCheck/tree/master/fonts and licensed under MIT by Roel Nieskens and Google.
+
+RanaKufi-Regular.subset.otf is from https://github.com/alif-type/rana-kufi/ and licensed under AGPLv3
```
Comment (ebraminio) [inline test/api/fonts/README]: Is it okay to use specially license-wise

File: test/api/test-ot-glyph.c [hunk 2]
```diff
@@ -158,6 +158,31 @@ test_hb_ot_glyph_cff1 (void)
...
+    .str = str,
+    .size = sizeof (str),
+    .consumed = 0
+  };
+  g_assert (hb_ot_glyph_decompose (font, 1, funcs, &user_data));
+  char expected[] = "M775,400C705,400 650,343 650,274L650,250L391,250L713,572L392,893"
+		    "L287,1000C311,942 296,869 250,823C250,823 286,858 321,823L571,572"
+		    "L150,150L750,150L750,276C750,289 761,300 775,300C789,300 800,289 800,276"
+		    "L800,100L150,100C100,100 100,150 100,150C100,85 58,23 0,0L900,0L900,274"
+		    "C900,343 844,400 775,400Z";
```
Comment (ebraminio) [inline test/api/test-ot-glyph.c]: Reverted the path to see effect of it to the test and have checked the path on browser to see if is ok.

File: test/api/fonts/README (hunk 1 above)
Comment (khaledhosny) [reply test/api/fonts/README]: Feel free to use that file under any suitable license. I didn't check this particular test, but I’m using the latest code and didn't notice any issue.

Comment (ebraminio) [reply test/api/fonts/README]: Thank you :)

**Pull request merged**